### PR TITLE
Post: hacktoberfest 2021

### PR DESCRIPTION
**Pull request by vepadulano**

### CODE REVIEW

**Review by Axel-Naumann — APPROVED** [review]
Maybe add "😅" after the last sentence?

File: blog/_posts/2021-10-05-hacktoberfest.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+layout: archive
+author: Vincenzo Eduardo Padulano
+---
+
+The ROOT GitHub repository is open for [Hacktoberfest](https://hacktoberfest.digitalocean.com/) contributions!
+
+## What is it?
+
+Hacktoberfest is a yearly event that encourages participation in open source communities and projects. Would you like to
+help us with some (not so scary) bugs? Or maybe you have some place in our documentation that you think deserves some
```
Comment (Axel-Naumann) [inline blog/_posts/2021-10-05-hacktoberfest.md:12]: Consider linking to "good first issue": https://github.com/root-project/root/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

**Comment by vepadulano** [comment]
> Maybe add "sweat_smile" after the last sentence?

Ok I thought it was enough writing `:sweat_smile:` on the markdown, but the rendered page doesn't show it correctly. How can I do this? https://root.cern/692/blog/hacktoberfest-2021/

**Comment by vepadulano** [comment]
Ok the character is now properly rendered, merging!

**Pull request merged**